src/playbook.yaml

```diff
@@ -54,4 +53,9 @@
     - name: Install CFN helper scripts
       ansible.builtin.pip:
         name: file:///tmp/aws-cfn-bootstrap-py3-latest.tar.gz
-        extra_args: "--target=/opt/aws/bin"
+
+    - name: Create CFN helper symbolic link
+      ansible.builtin.file:
+        src: "/usr/local/init/ubuntu/cfn-hup"
+        dest: "/etc/init.d/cfn-hup"
```

Comment:
I'm confused about how this interacts with https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/EC2/jc-ec2-linux.j2, will the file contents defined there overwrite this symlink when we boot an instance from the image?